internal/pkg/mapper/orchestrator/local_load.go

```diff
@@ -0,0 +1,386 @@
+package orchestrator
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/iancoleman/orderedmap"
+	"v.io/x/lib/toposort"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func (m *orchestratorMapper) OnObjectsLoad(event model.OnObjectsLoadEvent) error {
```

Comment:
`local load` orchestratora, ... obsahuje nejaky duplicitny kod s `remote load` (sorting, ..), to fixnem v dalsom PR, aby to bolo prehladnejsie.